bin/git-ignore-io

```diff
@@ -2,9 +2,13 @@
 
 gitignore_io_url="https://www.gitignore.io/api/"
 default_path="$HOME/.gi_list"
+if [[ -n $XDG_CACHE_HOME ]]; then
```

Comment:
Please quote the variables. The path might contains space.

Reply:
Done